pkg/plugin/packetparser/packetparser_linux.go

```diff
@@ -208,14 +213,17 @@ func (p *packetParser) Start(ctx context.Context) error {
 		p.callbackID = ps.Subscribe(common.PubSubEndpoints, &fn)
 	}
 
-	// Attach to eth0 for latency.
-	eth0Link, err := utils.GetInterface(Eth0, Device)
-	// eth0Link, err := retinautils.GetInterface("azvb22061c2658", "veth")
+	outgoingLinks, err := utils.GetDefaultOutgoingLinks()
 	if err != nil {
 		return err
 	}
-	p.l.Info("Attaching Packetparser to eth0", zap.Any("eth0Link", eth0Link.Attrs()))
-	p.createQdiscAndAttach(*eth0Link.Attrs(), Device)
+	if len(outgoingLinks) == 0 {
+		return errNoOutgoingLinks
+	}
+	outgoingLink := outgoingLinks[0] // Take first link until multi-link support is implemented
+
+	p.l.Info("Attaching Packetparser", zap.Any("outgoingLinks", outgoingLink.Attrs()))
```

Comment:
`zap.Any("outgoingLinks", outgoingLink.Attrs())` yields `unsupported value type`, can you pick which attributes you want to log?